tools/bazel.rc

```diff
@@ -1,7 +1,9 @@
 # This is so we understand failures better
 build --verbose_failures
 test --test_output=errors
+test --test_size_filters=-large,-enormous
 
-build --workspace_status_command=bin/get_workspace_status --action_env=ISTIO_VERSION
-
+build --workspace_status_command=bin/get_workspace_status --action_env=ISTIO_VERSION --output_filter=^gogo
```

Comment:
Consider adding a comment on which component needs the output_filter. i think it is Mixer

Reply:
care to educate what it does and why it's needed?

Reply:
Mixer has some extra paths mentioned in the gogo proto library build rule. For example `../../api`, `../api`, `api` in order for protos living outside Mixer repo to compile correctly. If the path is not used, gogo issues warning and this suppression is for not showing those warning about `path ../../api not used...`. I think we have a TODO or may be a bug to get rid of this kind of multiple paths listing.

Reply:
thanks! yeah, I know this warning.